### PR TITLE
support void type of function

### PR DESCRIPTION
**Pull request by YSawc**
modify support void type of function. I noticed that only reference assignment function that does not returns value is not supported yet.

### CODE REVIEW

**Comment by dalance** [comment]
Thank you for your contribution.
I think there is another solution to support void function.

```
// void return type support
function Func () -> void {
}

// omittable return type
function Func () {
}
```

How do you think?

**Comment by YSawc** [comment]
It sounds good! May i change emitter to apply your suggestion?

**Comment by YSawc** [comment]
@dalance 
[cecf6b7](https://github.com/dalance/veryl/pull/493/commits/cecf6b7abee8d2815735438e4d746e650eff1954)
I changed to apply omit return type. Please confirm changed logic.

File: crates/emitter/src/aligner.rs [hunk 1]
```diff
@@ -776,7 +776,10 @@ impl VerylWalker for Aligner {
         if let Some(ref x) = arg.function_declaration_opt0 {
             self.port_declaration(&x.port_declaration);
         }
-        self.minus_g_t(&arg.minus_g_t);
+        if let Some(ref x) = arg.function_declaration_opt1 {
+            self.minus_g_t(&x.minus_g_t);
+            self.scalar_type(&x.scalar_type);
```
Comment (dalance) [inline crates/emitter/src/aligner.rs:781]:
```suggestion
            // skip type align
            //self.scalar_type(&x.scalar_type);
```

File: crates/emitter/src/aligner.rs [hunk 2]
```diff
@@ -776,7 +776,10 @@ impl VerylWalker for Aligner {
...
+        }
         // skip type align
         //self.scalar_type(&arg.scalar_type);
```
Comment (dalance) [inline crates/emitter/src/aligner.rs:784]:
```suggestion
```

File: crates/emitter/src/aligner.rs (hunk 1 above)
Comment (dalance) [reply crates/emitter/src/aligner.rs:781]: The `scalar_type` should be skipped.
This causes the following unexpected spaces.

```
function automatic logic        [ParamX-1:0] FuncA(
                         ^^^^^^^
        input  logic        [ParamX-1:0] a,
                     ^^^^^^^
```

**Review by dalance — COMMENTED** [review]
Looks good!
I've added a comment about alignment of emitter.
Could you check it?

**Comment by YSawc** [comment]
Thanks your feedback! I applied each suggestions.

**Comment by dalance** [comment]
Thanks!!
I'll merge this PR.

**Pull request merged**